mozdownload/scraper.py

```diff
@@ -892,7 +891,13 @@ def get_build_info_for_index(self, build_index=None):
 
         # If no index has been given, set it to the last build of the day.
         if build_index is None:
-            build_index = len(parser.entries) - 1
+            # Find the most recent non-empty entry.
+            build_index = len(parser.entries)
+            for build in reversed(parser.entries):
```

Comment:
Thanks for the explanation. It makes perfect sense now. I find it helpful to see how other people operate. Gives me a different outlook on problems to solve in the future.

Cheers :)